### ISSUE TITLE
[Bug] [CPP RPC] custom_addr not properly quoted in JSON string

### ISSUE THREAD

**Issue by lhez** [apache/tvm#9174]
I happened to need `custom_addr` option in cpp rpc. However, `custom_addr_` does not seem properly quoted in the JSON string that is sent to the tracker, resulting in JSON parsing failure in the tracker.

In particular, there are 3 occurrences in `apps/cpp_rpc/rpc_tracker_client.h` - [line 88](https://github.com/apache/tvm/blob/main/apps/cpp_rpc/rpc_tracker_client.h#L88), [line 116](https://github.com/apache/tvm/blob/main/apps/cpp_rpc/rpc_tracker_client.h#L116) and [line 168](https://github.com/apache/tvm/blob/main/apps/cpp_rpc/rpc_tracker_client.h#L168).

**Comment by areusch** [comment]
thanks for reporting @lhez, @rkimball will take a look

**Comment by rkimball** [comment]
You are correct. We need to add \" around the `custom_addr` since it is a string.

**Comment by rkimball** [comment]
Here is a quick fix https://github.com/apache/tvm/pull/9192
